tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -334,6 +334,7 @@ private void EnableWaf()
                 }
 
 #else
+                _usingIntegratedPipeline = null;
```

Comment:
This could be removed, being picky.

Reply:
fixed e9b5f81f570178a129d60b598d9ec96619d2a755